File: fake-store-quarkus/src/main/java/org/arielzarate/interfaces/error/ApiErrorResponse.java
package org.arielzarate.interfaces.error;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ApiErrorResponse {
    private String type;
    private String title;
    private Integer status;
    private String detail;
    private String instance;
}

File: api-fake-hexagonal/src/main/kotlin/com/arielZarate/fake_api_hexagonal/interfaces/error/ApiErrorResponse.kt
package com.arielZarate.fake_api_hexagonal.interfaces.error

data class ApiErrorResponse(
    val type: String? = null,     // URL o identificador del tipo de error
    val title: String,            // Título legible del error
    val status: Int,              // Código HTTP
    val detail: String?,          // Detalle opcional
    val instance: String? = null  // URI de la solicitud que causó el error
) {
}